styles.css

```diff
@@ -1,33 +1,54 @@
-:root {
-    --menu-bar-width: 60px;
-    --menu-bar-height: 10px;
-}
-
-#menu {
-    background-color: #D84282;
-    position: relative;
-    width: var(--menu-bar-width);
-    height: var(--menu-bar-height);
-    margin: 100px;
-}
-
-#menu::before,
-#menu::after {
-    background-color: #D84282;
-    display: block;
-    content: "";
-    width: var(--menu-bar-width);
-    height: var(--menu-bar-height);
-    position: absolute;
-}
-
-#menu::before {
-    left: 0;
-    top: -20px;
-}
-
-#menu::after {
-    left: 0;
-    top: 20px;
-}
-
+:root {
+    --menu-bar-width: 60px;
+    --menu-bar-height: 10px;
+}
+
+#container {
+    background-color: white;
+    border: aqua 10px solid;
+    padding: 20px 0;
+    position: absolute;
+}
+
+#menu {
+    background-color: #D84282;
+    position: relative;
+    width: var(--menu-bar-width);
+    height: var(--menu-bar-height);
+}
+
+#menu::before,
+#menu::after {
+    background-color: #D84282;
+    content: "";
+    width: var(--menu-bar-width);
+    height: var(--menu-bar-height);
+    position: absolute;
+}
+
+#menu::before {
+    left: 0;
+    top: -20px;
+}
+
+#menu::after {
+    left: 0;
+    top: 20px;
+}
+#container:hover #menu {
+    background-color: white;
+}
+
+#container:hover #menu::before {
+    transform: translateY(20px) rotate(45deg);

```

Comment:
@christdoesselaere: de top parameter van "#container:hover #menu::before " is niet gespecificeerd (zou top: 0;) moeten zijn, maar zie dat je het hebt opgelost in de transform-waarde translateY.